test/test.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: CC0-1.0
+
+import unittest
```

Comment:
Let's not use unittest, it's an unproductive java-inspired monstrosity.

Just drop the `class` boilerplate and pytest will pick up the test with any issue.